openslides_backend/action/actions/mediafile/upload.py

```diff
@@ -31,25 +32,71 @@
 
 
 @register_action("mediafile.upload")
-class MediafileUploadAction(
-    CreateActionWithDependencies,
-    CreateActionWithListOfSpeakersMixin,
-):
+class MediafileUploadAction(MediafilePermissionMixin, CreateAction):
     """
     Action to upload a mediafile.
     """
 
     model = Mediafile()
     schema = DefaultSchema(Mediafile()).get_create_schema(
-        required_properties=["title", "meeting_id", "filename"],
-        optional_properties=["access_group_ids", "parent_id"],
+        required_properties=["title", "owner_id", "filename"],
+        optional_properties=["token", "access_group_ids", "parent_id"],
         additional_required_fields={"file": {"type": "string"}},
     )
     permission = Permissions.Mediafile.CAN_MANAGE
 
-    dependencies = [ListOfSpeakersCreate]
+    @original_instances
+    def get_updated_instances(self, action_data: ActionData) -> ActionData:
+        tokens: List[Any] = []
+        for instance in action_data:
+            collection, _ = self.get_owner_data(instance)
+            if collection != "organization":
+                continue
+            tokens.append(instance.get("token"))
+            results = self.datastore.filter(
+                self.model.collection,
+                FilterOperator("token", "=", instance["token"]),
```

Comment:
also add the owner_id filter here (see above)

Reply:
Done.